dev_guide/app_tutorials/maven_tutorial.adoc

```diff
@@ -78,6 +78,11 @@ $ oc set probe dc/nexus \
 	--failure-threshold 3 \
 	--initial-delay-seconds 30 \
 	-- echo ok
+oc set probe dc/nexus \
+	--readiness \
+	--failure-threshold 3 \
+	--initial-delay-seconds 30 \
+	--get-url=http://:8081/content/groups/public
```

Comment:
i'd expect a more "typical" liveness vs readiness pair to be:
1) liveness checks that the app server is still running (listening on port 80 and accepting connections)
2) readiness checks that the app itself is available/up/initialized

i don't think checking that the container is still live serves much purpose, if the process inside the container is hung.